product_development.md

```diff
@@ -0,0 +1,118 @@
+The product and design team owns the requirements engineering and conception of new features.
+They are responsible for creating tickets that are ready to be pulled into the backlog.
+
+### Growth opportunity
+
+Anything that can help a department (not only product related) to improve their performance and the performance of our business.
+They should be accompanied by metrics, their current state, and the projected state after the implementation of the growth opportunity.
+
+E.g: Feature A will boost KPI B from 10% to 20% and will increase revenue by 10%.
+
+Growth opportunities should align with our long term product vision.
+
+Depending on the complexity those opportunities might be developed in small increments prioritizing value delivery.
+
+Growth opportunity are owned by a stakeholder and prioritized via the stakeholder meeting.
+The product team is responsible to ensure that the ticket is ready to be pulled into the backlog.
+
+### Operational hiccups
+
+Operational hiccups are issues that disturb a stakeholders operation or hinder their ability to exercise their responsibilities.
+They usually require some kind of technical support or facilitation and should have some level of urgency similar to bugs.
+
+Operational hiccups are owned by a stakeholder and prioritized via the stakeholder meeting.
+The product team is responsible to ensure that the ticket is ready to be pulled into the backlog.
+
+### Bugs
+
+Bugs are owned by the engineering team as they affect the platform's stability.
+
+Bugs must have some kind of urgency. They are not prioritized by product, but triaged by engineering, based on what bug will kill our business first. Triage is handled by a designated senior engineer. Bugs are assigned to an engineer at all times but can be reassigned.
+
+A bug ticket should have an average resolution rate of less than a week and no more than two weeks.
+
+Bugs may be closed or converted into a technical dept ticket, if there is no longer a high urgency and no impact on the stability of the platform.
+Updates on bug tickets, especially closure without resolution, need to be communicated with product proactively.
+
+### Tech debt
+
+Engineering leads are responsible for the stability of the platform and the development velocity, and consequently for when and what technical debt needs to be handled.
+
+Work on technical debt must always be proactively communicated with the product team and documented in a ticket. Tech debt tickets must be pulled into the backlog in a refinement meeting, as they bind resources and take up significant space in the engineering backlog.
+
+### Platform announcement Slack channel
+
+Product, Design, Engineering, and Content team members post every change to go live on the platform into the Slack channel [#platform-announcements](https://voiio.slack.com/archives/C02BXK6R16V).
+This channel serves as a changelog for all employees and should include ever change the platform's experience, e.g.
```

Comment:
... and should include **_every_** change **_that affects_** the platform's experience ...

Reply:
```suggestion
This channel serves as a changelog for all employees and should include every change to the platform's experience, e.g.
```